Jenkinsfile

```diff
@@ -431,6 +439,18 @@ def runBuildAndroidApp() {
     }
 }
 
+def packForXcode() {
+    try {
+        sh """
+            cd examples/kmm-sample
+            ./gradlew :shared:packForXcode --stacktrace --no-daemon
+        """
+    } catch (err) {
```

Comment:
Can't really recall why we have this pattern and not just let the error propagate. Wouldn't it cause the pipeline to keep on building but just force the overall build to fail?